var/spack/repos/builtin/packages/py-cython/package.py

```diff
@@ -11,6 +11,8 @@ class PyCython(PythonPackage):
     homepage = "https://pypi.python.org/pypi/cython"
     url      = "https://pypi.io/packages/source/c/cython/Cython-0.25.2.tar.gz"
 
+    version('0.29', '5cb227772fbb0318be3c73b532d249ac')
```

Comment:
could you use the sha256 hashes?

Reply:
```suggestion
    version('0.29', sha256='94916d1ede67682638d3cc0feb10648ff14dc51fb7a7f147f4fedce78eaaea97')
```

Reply:
@Sinan81 sha256 hashes are not committed (or at least I can't see them).